docs/deploying/upgrade.md

```diff
@@ -24,3 +25,22 @@ lakefs migrate up
 Deploy (or run) the new version of lakeFS.
 
 Note that an older version of lakeFS cannot run on a migrated database.
+
+
+# Migrating from lakeFS < 0.30.0
+
+Starting version 0.30.0, lakeFS handles your committed metadata in a new way, which is more robust and has better performance.
+To move your existing data, you will need to run the following upgrade command with the updated lakefs binary:
```

Comment:
perhaps verify first that you're running a binary that is == 0.30.x with `lakefs --version`